ui/src/message_popup/playtest/rosa_experience_page.jsx

```diff
@@ -0,0 +1,132 @@
+/* @flow weak */
+import React from 'react';
+import uuid from 'uuid';
+
+import * as Api from '../../helpers/api.js';
+import hash from '../../helpers/hash.js';
+import LinearSession from '../linear_session/linear_session.jsx';
+import SessionFrame from '../linear_session/session_frame.jsx';
+import IntroWithEmail from '../linear_session/intro_with_email.jsx';
+
+import QuestionInterpreter from '../renderers/question_interpreter.jsx';
+import type {QuestionT} from './pairs_scenario.jsx';
+import RosaScenario from './rosa_scenario.jsx';
+import AudioResponseSummary from '../renderers/audio_response_summary.jsx';
+
+type ResponseT = {
+  choice:string,
+  question:QuestionT
+};
+
+
+
+// This is a scenario around bubble sort and classroom management.
```

Comment:
Since this is exactly the same as Jayden (just different questions), and is likely to stay the same overall structure, this could be a good abstraction to make - something like a `CounselingScenario` component that just takes the set of questions to use.  Then the `JaydenExperiencePage` and `RosaExperiencePage` both call that but provide different sets of questions.

If you're into that, you could do that in this pull request, or it's fine to leave this as its own file that's copy-pasted for now too.